client/src/templates/Challenges/utils/worker-executor.js

```diff
@@ -0,0 +1,45 @@
+export default class WorkerExecutor {
```

Comment:
Can we have some tests on this? We only need to test the 'public' API of this Worker, i.e. `sass` in, `css | Error.message` out.